go/host/l1/publisher.go

```diff
@@ -286,10 +286,6 @@ func (p *Publisher) PublishCrossChainBundle(bundle *common.ExtCrossChainBundle)
 		return nil
 	}
 
-	if len(bundle.CrossChainRootHashes) == 0 {
```

Comment:
am I missing something, or will this keep publishing every time?

Reply:
the guardian loop continues before calling this when empty

Reply:
what line?

Reply:
658
<img width="523" alt="image" src="https://github.com/ten-protocol/go-ten/assets/46542846/4f8e026e-8221-43be-819e-62a082ff2cdb">